source/Cosmos.HAL2/BlockDevice/ATAPI.cs

```diff
@@ -264,5 +272,44 @@ private void CheckForErrors()
                 //throw new Exception("ATAPI DRQ not set");
             }
         }
+        /// <summary>
+        ///  The function is chack the driver for Max LBA I took it from https://forum.osdev.org/viewtopic.php?f=1&t=14604
```

Comment:
Rephrase. "The function returns the max LBA value of the ATAPI device. Code is based on https://forum.osdev.org/viewtopic.php?f=1&t=14604"

Reply:
I Rephrased as reqested